vladislavVVVVV/SOA
# Add auth jwt tokens

Add authentication with  jwt tokens
Test plan: 
- drop previous data base `dotnet ef database drop`
- run `dotnet ef database update`
- run `dotnet watch run`
- see next screen
![Снимок экрана 2021-04-13 в 02 52 44](https://user-images.githubusercontent.com/49831502/114476660-53d16980-9c03-11eb-9822-d9d4de24b4cd.png)
- add any new user
- get auth token
- use postman or front api call with auth parameters
![Снимок экрана 2021-04-13 в 02 42 06](https://user-images.githubusercontent.com/49831502/114476741-81b6ae00-9c03-11eb-9b08-5a620eb936c1.png)
- try get access to any data
- make sure that any action with data require token
